README.md

```diff
@@ -39,6 +39,48 @@ elastic:
   caCertPath: "my/path/ca" #optional, path of custom CA file (it may be needed in some HTTPS connection..)
 ```
 
+There are some additional features Mr Plow can manage, such as specifying the expected data type to be inserted in ElasticSearch JSON (not specified fields are parsed as String):
+
+```yaml
+pollingSeconds: 5
+database: databaseValue
+queries:
+  - index: index_1
+    query: select * from employees
+    updateDate: last_update
+    fields:
+      - name: name
+        type: String
+      - name: working_hours
+        type: Integer
+```
+
+And additionally (for database like Postgres) you can also specify JSON columns fields:
```

Comment:
I would like to split the example below in two examples:
1. just json parsing
2. json parsing + casting (since casting is not mandatory, it may be confusing for new users.. and they may think that they need to list all the names of the json also without casting)

Reply:
Done, this should be ok for me